Casks/b/balsamiq-wireframes.rb

```diff
@@ -9,7 +9,16 @@
 
   livecheck do
     url "https://builds.balsamiq.com/bwd/mac.jsonp"
-    regex(/"version"\s*:\s*"(\d+(?:\.\d+)+)"/i)
+    regex(/callback\((.+?)\);/i)
+    strategy :page_match do |page, regex|
+      match = page.match(regex)
+      next if match.blank?
+
+      json_string = match[1]
+
+      json = Homebrew::Livecheck::Strategy::Json.parse_json(json_string)
```

Comment:
```suggestion
      json = Homebrew::Livecheck::Strategy::Json.parse_json(match[1])
```

We only use `json_string` once and we know that `match[1]` will exist at this point (since it's a non-optional capture group in the regex), so we can simply inline this instead of creating a variable.